vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/MultiSelectComboBox.java

```diff
@@ -561,7 +559,7 @@ public void setI18n(MultiSelectComboBoxI18n i18n) {
      * settings of the web component.
      */
     private void updateI18n() {
-        JsonObject i18nJson = (JsonObject) JsonSerializer.toJson(this.i18n);
+        JsonObject i18nJson = (JsonObject) JsonSerializer.toJson(getI18n());
```

Comment:
Shouldn't `requiredErrorMessage` be removed from the serialized `i18n` object as the web component does not use it?

Reply:
Good catch, I missed this.

Reply:
Fixed.